### PR TITLE
Add geocomputing to tutorials

### PR DESCRIPTION
**Pull request by ktiits**
## Proposed changes

Briefly describe the changes you've made here. Remember to add a link to the [preview page](https://csc-guide-preview.rahtiapp.fi/origin/) of your branch.

## Checklist before requesting a review

- [ ] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [ ] My pull request passes all tests.
- [ ] I have included a link to the appropriate [preview page](https://csc-guide-preview.rahtiapp.fi/origin/) (select your branch from the list).

### CODE REVIEW

File: docs/support/tutorials/index.md [hunk 1]
```diff
@@ -57,10 +57,12 @@
 * [Starting with parallel R](parallel-r.md)
 
 ## Geoinformatics
+* [Geocomputing in CSC computing environment](https://research.csc.fi/geocomputing)
```
Comment (samumantha) [inline docs/support/tutorials/index.md:60]: Was the intention to link this one maybe: https://csc-training.github.io/geocomputing_course/ ? The currently linked page is no tutorial :thinking:

File: docs/support/tutorials/index.md [hunk 2]
```diff
@@ -57,10 +57,12 @@
...
 * [Earth Observation (EO) guide](gis/eo_guide.md)
 * [Using geospatial files directly from cloud, inc Allas](gis/gdal_cloud.md)
 * [GDAL tutorial](gis/gdal.md)
 * [Virtual rasters](gis/virtual-rasters.md)
+* For software specific tutorials, see [the respective software pages](../../apps/by_discipline.md#geosciences)
```
Comment (samumantha) [inline docs/support/tutorials/index.md:65]:
```suggestion
* For software specific tutorials and examples, see [the respective software pages](../../apps/by_discipline.md#geosciences)
```
Not every page has a tutorial?

Comment (ktiits) [reply docs/support/tutorials/index.md:65]: Not every page has them, but the main ones (Python and R) have rather nice listings there also...

File: docs/support/tutorials/index.md (hunk 1 above)
Comment (ktiits) [reply docs/support/tutorials/index.md:60]: That is a little bit matter of taste. For example the DA-guide is not much different. I would like to include the link here, could think of having then some better describing link text..

Comment (samumantha) [reply docs/support/tutorials/index.md:60]: Yes updating the link text might help

Comment (ktiits) [reply docs/support/tutorials/index.md]: Changed the link text.

**Review by EetuHuuskoCSC — APPROVED** [review]
LGTM

**Pull request merged**